dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/TextAreaParam.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { IBasicFormParam } from "shared/types";
+
+export interface IStringParamProps {
+  id: string;
+  label: string;
+  param: IBasicFormParam;
+  handleBasicFormParamChange: (
+    p: IBasicFormParam,
+  ) => (e: React.FormEvent<HTMLTextAreaElement>) => void;
+}
+
+class TextAreaParam extends React.Component<IStringParamProps> {
```

Comment:
Instead of creating a new Component I would reuse the existing `TextParam`. These two components are mostly the same, the only different is that `TextParam` uses `<input` and this one uses `<textarea`. We can save a lot of code if we reuse the `TextParam`, you just need to filter, for example using the `inputType` parameter there. Something like:

```
let input = <input ...
if (inputType === "textarea") {
 input = <textarea ...
}
    return (
...
            <div className="col-9 margin-t-small">
              {input}
...
```